vignettes/adsl.Rmd

```diff
@@ -372,6 +373,42 @@ This call would return the input dataset with the column `RANDDT` is added.
 dataset_vignette(adsl, display_vars = exprs(USUBJID, RANDDT))
 ```
 
+## Derive Birth Date and Analysis Age (`BRTHDT`, `AAGE`, `AAGEU`) {#aage}
+
+The function `derive_vars_aage()` can be used to derive analysis age (`AAGE`) and 
+analysis age unit (`AAGEU`). The function derives the age based on the birth date 
+(`BRTHDT`) and a reference date, which is typically the randomization date (`RANDDT`).
+
+Note that `BRTHDT` must be derived first from `BRTHDTC` using `derive_vars_dt()`.
+
+```{r eval=TRUE}
+# Derive birth date from BRTHDTC
+adsl <- adsl %>%
+  derive_vars_dt(
+    new_vars_prefix = "BRTH",
+    dtc = BRTHDTC
+  )
+```
+
+Typically, dates of birth are collected only as years. However, this data has complete birth dates. The function `derive_vars_dt()` has the flexibility to do imputation on partial dates. Please see the `derive_vars_dt()` documentation and examples for more details or check out [Derive Death Variables](#death) section below.
```

Comment:
I think the text in the "Derive Death Variables" section is not up to date:
> This call would return the input dataset with the columns DTHDT added and, by default, the associated date imputation flag (DTHDTF) populated with the controlled terminology outlined in the ADaM IG for date imputations. If the imputation flag is not required, the user must set the argument flag_imputation to "none".

The call doesn't impute. Thus no imputation flag is created. We should update it and mention that the imputation flag is created automatically if it is required.